src/views/partials/open_data/template.njk

```diff
@@ -0,0 +1,29 @@
+{% extends 'partials/modal/template.njk' %}
+
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+
+{% block modalContent %}
+  <div class='data-container'>
+    <h1 class='govuk-heading-l data-container__heading-l govuk-!-margin-bottom-0'>You are about to leave Natural Capital Search Service</h1>
+    <p class='govuk-body-l data-container__body-l govuk-!-margin-bottom-0'>You are following a link to an external catalogue hosted by:"<span id='resource_locator'></span>"</p>
```

Comment:
by: is on the next line, but on the design it is in the same line. is that because of the modal width you have chosen?

Reply:
This was happen because of the space issue after `by:`. Now it has been fixed.